google/ggrc-core
# GGRC-7766 Unify font style in notifications

GGRC-7766 Unify font style in notifications
# Issue description

Font sizes in notifications was different because of wrong html tag in jinja-template

# Steps to test the changes

1. Open any Audit
2. Create Assessment
3. Make any changes in Assessment e.g change Description
4. Run Daily digest (_notifications/show_daily_digest)
5. Review the font in the notification - font size for 'New assessments were created' and 'Assessment ... has been updated' should be equal

# Solution description

Change tag in jinja-template

# Sanity checklist



# PR Review checklist
